baseframe/forms/fields.py

```diff
@@ -305,39 +305,12 @@ def process_formdata(self, valuelist):
         self._timezone_converted = False
 
     def pre_validate(self, form):
-        if self._timezone_converted is False:
+        if self.data and self._timezone_converted is False:
```

Comment:
Incase DateTimeField is optional in the form, it was failing. So, I added the check. Do I remove it now and add it as different PR?